homeassistant/components/conversation/__init__.py

```diff
@@ -150,7 +152,6 @@ async def handle_reload(service: core.ServiceCall) -> None:
     hass.http.register_view(ConversationProcessView())
     websocket_api.async_register_command(hass, websocket_process)
     websocket_api.async_register_command(hass, websocket_prepare)
-    websocket_api.async_register_command(hass, websocket_get_agent_info)
```

Comment:
This one is actually used in the frontend to print the attribution in the assist dialog.